### DISCUSSION TITLE
Beats service monitoring

### DISCUSSION THREAD

**Discussion by m9o7h** [Security-Onion-Solutions/securityonion#7654] [Unsupported Versions] [answered]
How can I monitor the beat services that running on the remote machines?
I don't want to use the Osquery agent that can be managed by FleetDM, my beat agents are filebeat.

**Comment by dougburks** [comment, marked as answer]
If you expect those filebeat instances to be constantly sending logs, then one option might be to use Elastalert's flatline option to alert if no logs are received in a certain time period.
